### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by charleskawczynski**

### CODE REVIEW

**Comment by charleskawczynski** [comment]
bors try

File: examples/Emulator/GaussianProcess/Manifest.toml [hunk 1]
```diff
@@ -34,40 +103,58 @@ uuid = "83423d85-b0ee-5818-9007-b63ccbeb887a"
 version = "1.16.1+1"
 
 [[CalibrateEmulateSample]]
-deps = ["Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
-path = "/Users/tsj/Documents/climate/clima/CalibrateEmulateSample.jl"
+deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
+path = "../../.."
```
Comment (charleskawczynski) [inline examples/Emulator/GaussianProcess/Manifest.toml:107]: @odunbar this is what to look out for (and is the result of `dev`ing)

Comment (odunbar) [reply examples/Emulator/GaussianProcess/Manifest.toml:107]: Gotcha!

**Comment by odunbar** [comment]
See [here](https://github.com/CliMA/CalibrateEmulateSample.jl/pull/139/files#diff-551f9c068f7dfe55021873730347296eb06cf70d8f51d0bfcc46d820a248177cL135) to fix `test/GaussianProcess/runtests.jl` unit tests. it's just a case of too narrow tolerances when the random seed changed

That should cure the local tests.

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Comment by charleskawczynski** [comment]
bors try

**Pull request merged**